operations/app/terraform/vars/prod/locals.tf

```diff
@@ -2,15 +2,18 @@ locals {
   environment = "prod"
   address_id  = 7
   init = {
-    environment           = local.environment
-    location              = "eastus"
-    is_metabase_env       = true
-    resource_group_name   = "prime-data-hub-${local.environment}"
-    resource_prefix       = "pdh${local.environment}"
-    okta_redirect_url     = "https://prime.cdc.gov/download"
-    okta_base_url         = "hhs-prime.okta.com"
-    OKTA_scope            = "simple_report_dev"
-    storage_queue_name    = ["process"]
+    environment          = local.environment
+    location             = "eastus"
+    is_metabase_env      = true
+    resource_group_name  = "prime-data-hub-${local.environment}"
+    resource_prefix      = "pdh${local.environment}"
+    okta_redirect_url    = "https://prime.cdc.gov/download"
+    okta_base_url        = "hhs-prime.okta.com"
+    OKTA_scope           = "simple_report_dev"
+    RS_okta_base_url     = "reportstream.oktapreview.com"
```

Comment:
`RS_okta_base_url` is `"reportstream.okta.com"`

Reply:
RS_okta_base_url has been updated in the latest commit